apps/synapse-portal-framework/src/components/nf/FormModal.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react'
+import {
+  ConfirmationDialog,
+  DynamicForm,
+  SynapseComponents,
+} from 'synapse-react-client'
+import { Box } from '@mui/material'
+
+const FormModal = () => {
```

Comment:
This seems like a generally useful wrapper for DynamicForm!  Consider moving this to synapse-react-client, and adding the necessary FormModal props (union of DynamicForm props and submitButtonText).  Also consider renaming to DynamicFormModal